dlt/common/json/__init__.py

```diff
@@ -21,9 +21,28 @@
 
 TPuaDecoders = List[Callable[[Any], Any]]
 
+JsonSerializable = Union[str, Dict[str, Any]]
```

Comment:
This type is a bit naive. Possibly it would be better to use something slightly more accurate like described in https://github.com/python/typing/issues/182